src/scenes/end.js

```diff
@@ -74,12 +92,21 @@ export default class EndScene extends Phaser.Scene {
         this.add.text(
             this.game.config.width / 2 - 55,
             this.game.config.height / 2 - 30,
-            "Build your own",
+            strings.buildYourOwn,
             {
                 fontFamily: options.fontFamily,
                 fontSize: options.smallFontSize,
                 fill: options.purpleText,
             },
         );
     }
+
+    launchBuildYourOwn() {
+        window.location.href = "https://www.codecademy.com/learn/learn-phaser";
+    }
+
+    launchGame() {
+        this.scene.start("GameScene");
+        this.scene.stop("StartScene");
```

Comment:
I think it's largely inconsequential, but this should be `this.scene.stop('EndScene')`.

Reply:
Ha, copy pasta error on my part... nice catch!

Reply:
> I think it's largely inconsequential, but this should be `this.scene.stop('EndScene')`.